internal/app/singularity/cache_list_linux.go

```diff
@@ -31,82 +34,98 @@ func findSize(size int64) (string, error) {
 	return "", fmt.Errorf("failed to detect file size")
 }
 
-func listLibraryCache() error {
-	// loop through library cache
+// listLibraryCache will loop throught and list all library cache (~/.singularity/cache/library).
```

Comment:
loop **through**

Reply:
"list all elements in the ..."? "list all entries in the ..."?